lte/gateway/python/magma/enodebd/devices/device_utils.py

```diff
@@ -69,17 +72,27 @@ def get_device_name(
             return EnodebDeviceName.BAICELLS_RTS
         elif sw_version.startswith('BaiBS_RTSH_'):
             return EnodebDeviceName.BAICELLS_RTS
+        elif sw_version.startswith('BaiBS_QRTB_') \
+                and hw_version.startswith('E01') \
+                and product_class == 'FAP/mBS31001/SC':
```

Comment:
Define this as a constant

Reply:
We changed the detection code a little bit. However already existing detection code did not have constants for value comparison. We did not want to deviate from that approach.